ccmlib/scylla_docker_cluster.py

```diff
@@ -48,18 +50,21 @@ def _update_config(self):
 class ScyllaDockerNode(ScyllaNode):
     def __init__(self, *args, **kwargs):
         kwargs['save'] = False
+        # This line should appear before the "supper" method
+        self.share_directories = ['data', 'commitlogs', 'conf', 'hints', 'view_hints', 'saved_caches', 'keys']
         super(ScyllaDockerNode, self).__init__(*args, **kwargs)
         self.base_data_path = '/usr/lib/scylla'
         self.local_base_data_path = os.path.join(self.get_path(), 'data')
         self.local_yaml_path = os.path.join(self.get_path(), 'conf')
-        self.docker_name = f'{self.cluster.get_path().split("/")[-1]}-{self.cluster.name}-{self.name}'
+        folder_name = self.cluster.get_path().split("/")[-2]
```

Comment:
i became skeptic about this... should it really be `[-2]` or `[-1]`?
i saw when running from ccm cmd line it is not good with -2... locally i'm running it with -1 and it works fine (from ccm cmd line POV)

Reply:
Ill check it